src/store/modules/HardwareStatus/PcieTopologyStore.js

```diff
@@ -261,10 +261,12 @@ const PcieTopologyStore = {
                       const oneSlot =
                         oneChassis.detailedInfo.pcieSlots.eachSlot[y];
                       if (
+                        chassisMembers[index].PCIeSlots.Slots[j].Links?.Oem?.IBM
+                          ?.AssociatedAssembly &&
                         oneSlot.associatedAssemblyLink &&
                         oneSlot.associatedAssemblyLink ===
                           chassisMembers[index].PCIeSlots.Slots[j].Links?.Oem
-                            .IBM.AssociatedAssembly[0]['@odata.id']
+                            ?.IBM?.AssociatedAssembly['@odata.id']
```

Comment:
I don't know about how js script works, but just comparing the codes, 
I'm wondering why "?" is added to "Oem?.IBM?" which did not exist before.

Other than this, it seems good to me.

Reply:
"?" is added so that, It first checks if the object is present or not. If the object is not present, It just ignores without giving UI error and prevents our page from breaking. Eg: If we don't have "IBM" variable in the redfish response, then it doesn't break the page if we add "?".

Reply:
Thanks for explanation.